infra-as-code/bicep/storage.bicep

```diff
@@ -71,22 +82,34 @@ resource appDeployStorage 'Microsoft.Storage/storageAccounts@2023-05-01' = {
           enabled: true
           keyType: 'Account'
         }
+        queue: {
+          enabled: false
+          keyType: 'Account'
+        }
+        table: {
+          enabled: false
+          keyType: 'Account'
+        }
       }
     }
     minimumTlsVersion: 'TLS1_2'
     isHnsEnabled: false
     isSftpEnabled: false
-    // isLocalUserEnabled: false
-    // publicNetworkAccess: 'Disabled'
+    isLocalUserEnabled: false
+    publicNetworkAccess: 'Disabled'
     networkAcls: {
-      bypass: 'AzureServices'
+      bypass: 'None'
       defaultAction: 'Deny'
     }
     supportsHttpsTrafficOnly: true
   }
   resource blobService 'blobServices' = {
     name: 'default'
-
+    properties: {
+      containerDeleteRetentionPolicy : {
+        enabled: true
+      }
+    }

```

Comment:
Need to verify if this has any side effects for us for cleanup.  Probably doesn't, but just check to be sure.